code/modules/surgery/surgery.dm

```diff
@@ -553,3 +563,32 @@
 		for(var/reagent in chems_needed)
 			if(target.reagents.has_reagent(reagent))
 				return TRUE
+
+/datum/surgery_step/proc/play_preop_sound(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	if(!preop_sound || ismachineperson(target))
```

Comment:
```suggestion
	if(!preop_sound || (islist(preop_sound) && !length(preop_sound)) || ismachineperson(target))
```

Reply:
Is this to protect against empty lists trying to play a sound?

Reply:
Yeah, just a sanity check.